das/daser_test.go

```diff
@@ -17,7 +17,8 @@ import (
 
 	"github.com/celestiaorg/celestia-node/fraud"
 	"github.com/celestiaorg/celestia-node/header"
-	"github.com/celestiaorg/celestia-node/service/share"
+	"github.com/celestiaorg/celestia-node/nodebuilder/share"
```

Comment:
Daser should not import bodybuilder.

Reply:
What? Why? That is where the service definition is that we need.

Reply:
Ah, will try to cast to share.Availability then

Reply:
Yes. Thanks